firefox-ios/WidgetKit/OpenTabs/OpenTabsWidget.swift

```diff
@@ -37,6 +39,7 @@ struct OpenTabsView: View {
         let query = widgetFamily == .systemMedium ? "widget-tabs-medium-open-url" : "widget-tabs-large-open-url"
         VStack(alignment: .leading) {
             Link(destination: linkToContainingApp("?uuid=\(tab.uuid)", query: query)) {
+                /* Ecosia: Update default suggested sites entries
```

Comment:
Very minor, but can't we just comment out the first if line inside the completion and leave the rest out? I guess that would reduce conflicts.

Reply:
Holaaa! 
Not sure fully get it but I won't merge it as I do see it as a valuable comment, especially if that would help us reduce conflicts.
The way I managed it for this Widget and the one below is to comment on an entire block (and restore it to Firefox's code where needed) so that would be easier for us to replace blocks instead of multiple small portions of code throughout the completion's body. Of course, I tried to keep the amount of line changes at a minimum.

Reply:
Yeah, sorry for not being more clear, I could not suggest an edit on the whole code block 🙈 

But I see you already improved by moving it inside the completion, that's exactly what I meant 🙂 I guess it could be even further simplified by re-using the `.padding` as well as the last lines of the text attributes (`.multilineTextAlignment`, `.lineLimit` and `.font`) since those are duplicated inside the comment and outside currently.

Reply:
I updated it so it should reflect the intention now @lucaschifino

Reply:
Thanks! Changes look great!